schemaregistry/schemaregistry_client.go

```diff
@@ -212,7 +212,7 @@ type Client interface {
 	GetAllSubjects() ([]string, error)
 	DeleteSubject(subject string, permanent bool) ([]int, error)
 	DeleteSubjectVersion(subject string, version int, permanent bool) (deletes int, err error)
-	GetCompatibility(subject string) (compatibility Compatibility, err error)
+	GetCompatibility(subject string, defaultToGlobal bool) (compatibility Compatibility, err error)
```

Comment:
As discussed, this might break existing code because the function signature changes.

Reply:
@rayokota can you comment on that?